# Apply Wrench to the center of gravity of Skye.
# via the callback mechanism
# wrench expressed in Skye's local NED frame.
geometry_msgs/Wrench wrench               # wrench applied to the origin of the NED frame in the Cog
time start_time                           # (optional) wrench application start time (seconds)
                                          # if start_time is not specified, or
                                          # start_time < current time, start as soon as possible
duration duration                         # optional duration of wrench application time (seconds)
                                          # if duration < 0, apply wrench continuously without end
                                          # if duration = 0, do nothing
                                          # if duration < step size, apply wrench
                                          # for one step size
---
bool success                              # return true if set wrench successful
string status_message                     # comments if available
